docs/docs/plugins-themes-and-starters.md

```diff
@@ -85,23 +91,23 @@ Plugins and themes both allow options to be passed in when installed in the plug
 
 #### Shadowing
 
-The options exposed by a plugin or theme are up to the author. For example, a plugin or theme can allow a specific path to be provided in the `gatsby-config` so the plugin knows where to build pages from. In this case the user wouldn't be able adjust _how_ those pages are built, only from what path. As an escape hatch to change small pieces of an installed theme, [shadowing](/docs/themes/shadowing/) was introduced. Theme shadowing allows users to replace a file with their own version of it. Because themes are also plugins, shadowing is possible in plugins as well.
+As an escape hatch to change small pieces of an installed theme, [shadowing](/docs/themes/shadowing/) was introduced. For example, a plugin or theme could allow a specific path to be provided in the `gatsby-config` so the plugin knows where to build pages from, but the user wouldn't be able adjust _how_ those pages are built, only from what path. Theme shadowing allows users to replace a file with their own version of it, allowing them to rewrite that logic to use the path in a different way.
```

Comment:
```suggestion
Theme, [shadowing](/docs/themes/shadowing/) exists to allow users to override or otherwise extend individual components provided by a theme. For example, a plugin or theme can provide a specific path to `gatsby-config` so the plugin knows where to build pages from, but the user wouldn't be able adjust _how_ those pages are built, only from what path. Theme shadowing allows users to replace a file with their own version of it, allowing them to rewrite that logic to use the path in a different way.
```